buildSrc/src/main/kotlin/common-conventions.gradle.kts

```diff
@@ -0,0 +1,18 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+val deployVersion = project.findProperty("DeployVersion")
+if (deployVersion != null) project.version = deployVersion
+
+tasks.withType<KotlinCompile>().configureEach {
+    // Suppress the warning: 'expect'/'actual' classes (including interfaces, objects, annotations, enums, and 'actual' typealiases) are in Beta.
+    // See: https://youtrack.jetbrains.com/issue/KT-61573
+    compilerOptions { freeCompilerArgs.add("-Xexpect-actual-classes") }
+}
+
+plugins.withType<JavaPlugin> {
+    configure<JavaPluginExtension> {
+        toolchain {
+            languageVersion.set(JavaLanguageVersion.of(8))
+        }
+    }
+}
```

Comment:
These opt-in and toolchain conventions are specific to Kotlin subprojects.

I think it'd be better to do the following:

1. Move the KMP-specific expect/actual suppression to the KMP convention
2. Create a `kotlin-jvm` convention with the JVM toolchain config.

Reply:
fixed